cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,125 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+const faker = require('faker');
 
-describe('Article', () => {
-  before(() => {
+describe('Article user flow', () => {  
+beforeEach(() => {
+  cy.task('db:clear');
+  cy.visit('/#/login');
+});
 
-  });
+it('should be created using New Article form', () => {
+  let user;
```

Comment:
You could move login method to custom function, better to make it like an API function and decrease amount of code in your test